spec/support/shared_examples/signable_configuration.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+RSpec.shared_context 'when signable is configured' do
+  Signable.configure do |config|
+    config.base_url = 'api.signable.co.uk'
+    config.api_key  = ENV.fetch('SIGNABLE_API_KEY')
+  end
+end
```

Comment:
This gem obtains the base URL and security key for the Signable API from whatever app installs it, such as our `api` codebase. This context replicates that configuration in a test environment.